kpi/paginators.py

```diff
@@ -137,6 +137,13 @@ def get_count(self, queryset):
         return super().get_count(queryset)
 
 
+class OrganizationMembersPagination(PageNumberPagination):
+    """
+    Pagination class for Organization Members
+    """
+    page_size_query_param = 'page_size'
+
+
```

Comment:
You can delete this class, not needed anymore

Reply:
Done. It will use `LimitOffsetPagination` by default